assets/main.js

```diff
@@ -486,56 +542,59 @@ function multiselect(selected, options) {
 	};
 }
 
-async function updatePolicy() {
+async function updatePolicy(instanceName) {
```

Comment:
it's about style: you used `instance_name` previously, and usually use an `_` as separator -- and now there's camelcase `instanceName`...